articles/tutorial-template.md

```diff
@@ -0,0 +1,108 @@
+---
+title: Markdown template for quantum tutorials
+description: Template and guidance for writing tutorials for the Microsoft Quantum Development Kit documentation. 
```

Comment:
Make it clear that this meta description used for SEO, and that this is what is displayed on a search results page

Reply:
If you put the intent text in here, then we don't need to worry about the "As a content developer ..." stuff below.